cmd/minishift/cmd/openshift/set.go

```diff
@@ -78,6 +78,8 @@ func runPatch(cmd *cobra.Command, args []string) {
 	sshCommander := provision.GenericSSHCommander{Driver: host.Driver}
 	dockerCommander := docker.NewVmDockerCommander(sshCommander)
 
+	fmt.Println(fmt.Sprintf("Patching OpenShift configuration '%s' with '%s'", patchTarget.LocalConfigFile, patch))
```

Comment:
This should be behind Verbose 2.

Reply:
Also I am not sure why this Println is present here. Was it for debugging?

Reply:
Yes, this was for debugging.

Reply:
Nope, this is not for debugging, this println was present in the `pkg/openshift/openshift.go` i moved it to the cmd package, its a refactor